index.html

```diff
@@ -0,0 +1,720 @@
+                <li class="nav-one-li">Business</li>
+                <li class="nav-one-li">AMAs</li>
+                <li class="nav-one-li">Spaces</li>
+                <li class="nav-one-li">Term & Conditions</li>
+            </ul>
+        </div>
+
+        <div class="nav-two">
+            <img src="images/logo.png" alt="" class="logo">
+            <ul>
+                <li class="nav-two-li">Lates</li>
+                <li class="nav-two-li">Hard Fork</li>
+                <li class="nav-two-li">Plugged</li>
+                <li class="nav-two-li">Fundamentals</li>
+                <li class="nav-two-li">Work 2030</li>
+            </ul>
+            <button class="hamburger">
+                <i class="fas fa-bars"></i>
+            </button>
+            <button class="search"><i class="fas fa-search"></i></button>
+
+        </div>
+    </nav>
+    <main>
+        <header class="main">
+            <section class="header-blocks">
+                <article class="cover-story">
+                    <div class="cover-date">Saturday - July 20, 2019</div>
+                    <div class="cover-story-text">
+                        <h5>World</h5>
+                        <h2>After endless conflicts, an entrepreneurial ecosystem blooms in Iraq</h2>
+                        <ul>
+                            <li>by <a href="#">Ravie Lakshmanan</a></li>
+                            <li><i class="far fa-clock"></i>7 hours ago</li>
+                            <li><i class="fas fa-share-alt"></i>26 </li>
+                        </ul>
+                    </div>
+                </article>
+
+                <article class="cover-story">
+                    <div class="cover-story-text small-cover">
+                        <h2>CHEAP: Tell 360 degree tales of your adventures with $300 off GoPro Fusion</h2>
+                        <ul>
+                            <li>by <a href="#">Deals on TNW </a></li>
+                            <li><i class="far fa-clock"></i>7 hours ago</li>
+                            <li><i class="fas fa-share-alt">8</i></li>
+
+                        </ul>
+                    </div>
+                </article>
+
+                <article class="cover-story">
+                    <div class="cover-story-text small-cover">
+                        <h5>Insights</h5>
+                        <h2>US watchdog preps BitMEX probe over alleged illegal cryptocurrency trade</h2>
+                        <ul>
+                            <li>by <a href="#">Yessi Bello Perez</a></li>
+                            <li><i class="far fa-clock"></i>8 hours ago</li>
+                            <li><i class="fas fa-share-alt">1</i></li>
+                        </ul>
+                    </div>
+                </article>
+            </section>
+        </header>
+
+
+        <section class="latest-news">
+            <h2 class="latest-news-header">Latest news ></h2>
+
+            <div class="news-block">
```

Comment:
Makes these divs `article` tags instead. It is more appropriate.

Also you use `article` tags in other sections, it's best to make it uniform.

Reply:
Do this for all `news-block` divs in Latest news section. Make the `article` tags